libr/core/cmd_help.c

```diff
@@ -426,6 +426,10 @@ static int cmd_help(void *data, const char *input) {
 	case 'B': // "?B"
 		k = r_str_trim_ro (input + 1);
 		tmp = r_core_get_boundaries_prot (core, -1, k, "search");
+		if (!tmp) {
+			R_LOG_WARN ("Cannot get boundaries\n");
```

Comment:
why not put this warnin in r_core_get_boundaries_prot?

Reply:
because you don't always want to do this. Apparently there are cases where you have a fallback solution (e.g. in cmd_anal.c or in cmd_print.c)

Reply:
Ok i have removed all those R_LOG_WARN. NULL is returned only if allocation fails and we don't usually prints messages when allocation fails... should be good to merge i think (let's wait for travis though)